### PR TITLE
Add SP indicator+toggle to (P)LT screen

### PR DESCRIPTION
**Pull request by frank-weinberg**

### CODE REVIEW

File: html/controls/plt/plt-input.js [hunk 1]
```diff
@@ -24,16 +24,18 @@ function preparePltInputTable(element, teamId, mode, statsbookPeriod, alternateN
...
 			} else {
-				$('<td>').appendTo(thead);
+				$('<td>').text('Bench').appendTo(thead);
+				$('<td>').text('Jammer').appendTo(thead);
+				$('<td>').text('Pivot').appendTo(thead);
+				$('<td>').text('Blocker').appendTo(thead);
+				$('<td>').text('Box').appendTo(thead);
 			}
+			$('<td>').attr('id', 'StarPass').text('SP').click(function() {
+				WS.Set('ScoreBoard.Team('+teamId+').StarPass', !$(this).hasClass('Active'));
```
Comment (brian-brazil) [inline html/controls/plt/plt-input.js:37]: Can it be made more visually obvious that this is a button?

Comment (frank-weinberg) [reply html/controls/plt/plt-input.js:37]: Do you have a suggestion as to how (especially given that none of the other buttons on this screen is specifically marked as such)?

Comment (brian-brazil) [reply html/controls/plt/plt-input.js:37]: I was thinking either a different background, or putting round corners inside the td.

Comment (frank-weinberg) [reply html/controls/plt/plt-input.js:37]: I've gone for a blue background as no other button on the screen has round corners.

**Pull request merged**